web/src/lib/components/Avatar.svelte

```diff
@@ -23,13 +23,13 @@
                     <img src={account.profileImageUrl} alt="{account.nym}'s avatar" />
                 </div>
             </div>
-            <div class="font-bold flex items-center justify-center" class:text-2xl={size === AvatarSize.L}>
+            <!-- <div class="font-bold flex items-center justify-center" class:text-2xl={size === AvatarSize.L}>
```

Comment:
What is the reason for commenting this? This component is used all over the place... If you don't want to display the nym in *some* cases, perhaps you can just add some other flag, which hides it in the cases you want it hidden.